src/Aspire.Cli/Certificates/CertificateService.cs

```diff
@@ -50,11 +50,27 @@ public async Task EnsureCertificatesTrustedAsync(IDotNetCliRunner runner, Cancel
 
             if (trustExitCode != 0)
             {
+                var outputLines = ensureCertificateCollector.GetLines();
+
+                if (outputLines.Any(line => line.Line == DevCertsPartialTrustMessage))
```

Comment:
Wouldn't it be better to use the exit code?

Reply:
Updated code to use exit code. The exit code is not unique, there is a scenario where exit code 4 should be fatal, but also a scenario where we want it to be non-fatal so I use the output lines to disambiguate.